### DISCUSSION TITLE
Hide specific days since account was signed

### DISCUSSION THREAD

**Discussion by RequestPrivacy** [bisq-network/bisq#5525] [Ideas]
I was wondering if there is a need to show the exact days since an account was signed. I had occasions where multiple offers where available and all had the exact same account age, so an outsider could easily derive that they all belong to the same entity. Together with the amount of the offerings one could even guess the net worth of this individual.

For enhanced privacy I would change the visible (exact) account age to something rather unspecific like >200 days / >300 days. It is just a small privacy gain but I can't see a downside and the general information (was the account just created, has it a bit of history etc.) is still avialable.

**Comment by pazza83** [comment]
I do not think hiding the exact days since the wallet was signed would be helpful in the situation your describe. It is easy to see what offer belong to which entity as they are linked by the same onion address / avatar / avatar notes.

I do think a range of 100 days would give more privacy though.

**Reply by RequestPrivacy** [reply]
You're right, I haven't thought about the onion address. So in my first described case the procedure from @MwithM https://github.com/bisq-network/bisq/discussions/5525#discussioncomment-779716 would be indeed the most helpful one. 

Nevertheless, I for one do change my whole bisq wallet and the corresponding onion address from time to time for a clean new start - the only outside identifier that remains is the exact payment account age (which is rather specific due to the comparatively few users). To avoid the hassle of previously generating a second account and self sign it, the more general account age would be still beneficial imho.

The bigger the age spacing, the greater the crowd you're hiding in. So maybe the steps could be something like >30 days / >100 days / > 300days / >500 days?

**Reply by ripcurlx** [reply]
The problem with this is, that we publish the timestamp of the signing and account creation in the network, so even if we would adapt this in the UI, it still would be possible for a sophisticated user to access this information.

**Reply by RequestPrivacy** [reply]
Well, in that case it would be just cosmetically with barely a privacy gain, indeed. More something against casual onlookers. Maybe someone finds time to implement it tough - even for such a "minimal" benefit. So self sign & new onion address is the way to go.

Thanks for the discussion!

**Comment by MwithM** [comment]
Tip: Valid for SEPA, but it should apply to all payment methods using name and family name. Create the exact same account again with a different salt, [self sign it](https://bisq.wiki/Account_limits#With_your_own_signed_account), wait 30 days and start using it. For further privacy, generate a [new onion address](https://bisq.wiki/Changing_your_onion_address) and it will look like you're very new to Bisq to any network participants, but keeping your buying limits intact.
